frontend/src/common/constansts/notifications.ts

```diff
@@ -0,0 +1,3 @@
+export enum Notifications {
+    SIGN_UP_MESSAGE = "You have successfully registered"
```

Comment:
I don't think you need to add MESSAGE to a variable, because all of them are messages. I would leave just `SIGN_UP`